content/guides/embed/ui-elements/installation.md

```diff
@@ -140,7 +128,7 @@ Most of the UI elements have 2 distinct versions.
   not get bundled with React and ReactDOM.
   - Use this when both React and ReactDOM libraries are already loaded on the
     application.
-  - These libraries expect a React and ReactDOM `>= 16.2` and `< 17`.
+  - These libraries expect a React and ReactDOM `>= 16.6` and `< 18`.
```

Comment:
I tested the `[...]-no-react.html` pages in [BUIE](https://github.com/box/box-ui-elements/tree/master/test) and all React + ReactDOM versions >= 16.6 worked, even the latest versions of 18

but I still left the maximum as 18 since it's possible something could break in a future 18 release